python/ray/train/huggingface/transformers/_transformers_utils.py

```diff
@@ -230,13 +230,20 @@ def on_train_end(self, args, state, control, **kwargs):
 class RayTrainReportCallback(TrainerCallback):
     """A simple callback to report checkpoints and metrics to Ray Tarin.
 
-    This callback overrides the `TrainerCallback.on_save()` method. After
+    This callback is a subclass of `transformers.TrainerCallback
+    <https://huggingface.co/docs/transformers/main/en/main_classes/callback#transformers.TrainerCallback>`_
+    and overrides the `TrainerCallback.on_save()` method. After
     a new checkpoint get saved, it fetches the latest metric dictionary
     from `TrainerState.log_history` and reports it with the latest checkpoint
     to Ray Train.
 
+    After training finished, one can retrieve the original transformer's checkpoint
+    folder under `checkpoint_00000*/checkpoint`.
```

Comment:
Reading over this I don't think it's immediately obvious to the user what `checkpoint_00000*` is - what do you think about formatting it like this?

Checkpoints will be saved in the following structure:
```
checkpoint_00000*/    # Ray Train Checkpoint
└─ checkpoint/        # Hugging Face Transformers Checkpoint
```

Reply:
I agree with it and this └─ character does looks beautiful.